app/templates/_typeSimpleWebApp/_index.html

```diff
@@ -4,28 +4,24 @@
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <title><%= site_name %></title>
-    <link rel="stylesheet" href="css/master.css">
     <link rel="shortcut icon" type="image/x-icon" href="favicon.ico">
+
+    <!-- CSS files -->
+    <link rel="stylesheet" href="css/master.css">
     <link href='http://fonts.googleapis.com/css?family=Open+Sans:400,300,600' rel='stylesheet' type='text/css'>
 </head>
 <body>
+    @@include('./partials/_header.html')
```

Comment:
@rgksugan Simple web app is meant to as simple & less gulp plugins as possible. That's why I removed even partials from this app type.